data/tabular/nr_ppar_gamma_tox21/transform.py

```diff
@@ -63,20 +64,16 @@ def get_and_transform_data():
                     {
                         "noun": "peroxisome proliferator-activated receptor gamma toxicity"
                     },
-                    {"verb": "is active in the NR-PPAR-gamma toxicity assay"},
-                    {
-                        "verb": "is active in the NR-peroxisome proliferator-activated receptor gamma toxicity assay"
-                    },
-                    {
-                        "verb": "is active in the peroxisome proliferator-activated receptor gamma toxicity assay"
-                    },
                     {"adjective": "toxic in the NR-PPAR-gamma assay"},
                     {
                         "adjective": "toxic in the NR-peroxisome proliferator-activated receptor gamma assay"
                     },
                     {
                         "adjective": "toxic in the peroxisome proliferator-activated receptor gamma assay"
                     },
+                    {"gerund": "showing toxicity in the NR-PPAR-gamma assay"},
```

Comment:
Here you can but showing, displaying, and exhibiting like that into the text template:
`{#showing|displaying|exhibiting!}`
Like that you need only one gerund here.

Reply:
you mean I can do something like this?
```suggestion
                    {"gerund": "{#showing|displaying|exhibiting!} toxicity in the NR-PPAR-gamma assay"},
```

Reply:
Ah, sorry, I was unclear, the `{#showing|displaying|exhibiting!}` would need to move to the text template and the gerund then would look like that:
`{"gerund": "toxicity in the NR-PPAR-gamma assay"},`

But this functionality of using the text part sampling is a good idea! We can think about it in the next iteration.

Reply:
then it is not a gerund and just a noun

Reply:
yeah, I'm also fine with keeping it like that, in the end the result will be the same